app/src/main/java/uz/direction/threadshomework/MainActivity.kt

```diff
@@ -1,18 +1,132 @@
 package uz.direction.threadshomework
 
-import android.os.Bundle
+import android.os.*
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import uz.direction.threadshomework.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
     lateinit var binding: ActivityMainBinding
 
+    private val handler1 = object : Handler(Looper.getMainLooper()){
```

Comment:
It could be done with one handler and you could create threads with loop, code would be nicer

Reply:
"Each Handler instance is associated with a single thread and that thread's message queue." from https://developer.android.com/reference/android/os/Handler#:~:text=A%20Handler%20allows%20you%20to,is%20bound%20to%20a%20Looper%20 says that each handler should be associated with one thread, so how to associate one handler several threads?